README.md

```diff
@@ -6,183 +6,111 @@
-
-*version*: The version in the check_mk packages - for example if the RPM is
-'check_mk-agent-1.2.0p3-1.noarch.rpm' then the version is '1.2.0p3-1'.
-Only required if a filestore is used.
-
-*workspace*: The directory to use to store files used during installation.
-Default: '/root/check_mk'
-
-*mrpe_checks*: Specifies a hash of check_mk::agent::mrpe resources to create. Default: {}
-
-## Host groups and tags
-
-By default check_mk puts all hosts into a group called 'check_mk' but where you
-have more than a few you will often want your own groups.  We can do this by
-setting host tags on the agents and then configuring host groups on the server
-side to match hosts with these tags.
-
-For example in the hiera config for your agent hosts you could have:
-
-    check_mk::agent::host_tags:
-      - '%{osfamily}'
-
-and on the monitoring host you could have:
-
-    check_mk::host_groups:
-      RedHat:
-        description: 'RedHat or_CentOS hosts'
-        host_tags:
-          - RedHat
-      Debian:
-        description: 'Debian or Ubuntu_hosts'
-        host_tags:
-          - Debian
-      SuSE:
-        description: 'SuSE hosts'
-        host_tags:
-          - Suse
-
-You can of course have as many host tags as you like. I have custom facts for
-the server role and the environment type (dev, qa, stage, prod) and define
-groups based on the role and envtype host tags.
-
-Remember to run the Puppet agent on your agent hosts to export any host tags
-and run the Puppet agent on the monitoring host to pick up any changes to the
-host groups.
-
-## Static host config
-
-Hosts that do not run Puppet with the check_mk module needs to get added to hiera.
-check_mk::config creates the config file
-/omd/sites/monitoring/etc/check_mk/all_hosts_static from a template.
-The template will look for hiera variables.
-The hiera variable check_mk::all_hosts_static has to be an array:
-
-check_mk::all_hosts_static:
-  - host1.domain
-  - host2.domain
-
-
-You can also include host tags - for example:
-
-check_mk::all_hosts_static:
-  - host1.domain|windows|dev,
-  - host2.domain|windows|prod,
-
+## Description
+The module installs and configures the Open Monitoring Distribution (OMD) which includes Nagios, check_mk and lots of other tools. Beside the server component, the module is also able to install and configure the check_mk agent.
+
+Original coded by erwbgy, forked gnubila-france and finally taken over by Voxpupuli.
+
+## Setup
+The module has been tested with:
+  * CentOS 6 and 7;
+  * Debian 8 and 9;
+  * Puppet version 5 and Puppet 6;
+  * check_mk version 1.5.x.
+
+Also it requires the following forge modules:
+  * puppetlabs/concat
+  * puppetlabs/stdlib
+  * camptocamp/systemd
+
+## Usage
+### Server
+#### Basic usage
+The following code will install check_mk and create one 'omd site'. Once this is done you should be able to access it by visiting:
+
+```http://<IP of the machine>/monitoring/```
+
+Debian 9:
+```
```

Comment:
You could mark the example as puppet code.  ie use...
```
```puppet
```

Reply:
Done